tests/drivers/disk/disk_performance/boards/mimxrt1060_evk.overlay

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
did this test case work with rt1060 before adding revC?

Reply:
No, such case is filter out before. and then marked as no run, but actually it is stack overflow, which twister set this to not  runnable